### ISSUE TITLE
License unclear

### ISSUE THREAD

**Issue by dylancvdean** [Stability-AI/StableLM#77]
The base model is described in README.md as being licensed under CC BY-SA-4.0, but the description of the license instead matches CC BY-4.0, which is linked in 2/3 links in that section. The licensing section should either include the requirement under CC BY-SA-4.0 to release modified versions of the base model and to link to the correct license, or should be changed to the CC BY-4.0 license that more closely matches the description. Otherwise, the license is ambiguous.

**Comment by mcmonkey4eva** [comment]
The base model is licensed as [CC BY-SA-4.0](https://creativecommons.org/licenses/by-sa/4.0/)

The "SA" part, "Share-Alike", is a requirement (ie, you may not redistribute a derivative of StableLM that does not follow a matching license).

**Comment by AndrewGodfrey** [comment]
This is not fixed yet. The description of the base model still has 2 links that go to a different license.

**Comment by dylancvdean** [comment]
Not only that, it describes the SA obligation as only indicating if changes are made rather than distributing under the same license as well, and then that description links to the non SA license. This is very likely to confuse people unfamiliar with the CC license family, especially as this model attracts attention from hobbyists

**Comment by AndrewGodfrey** [comment]
Oh yes, that was very misleading as well. And even if this was clearly written, the meaning of “remix” and “redistribute” aren’t clear when applied to the inference results of a LLM.

**Comment by mcmonkey4eva** [comment]
Oh oh my god, I finally found what you're referring to, `Under the license, you must give [credit](https://creativecommons.org/licenses/by/4.0/#) to Stability AI, provide a link to the license, and [indicate if changes were made](https://creativecommons.org/licenses/by/4.0/#).` in the readme - the explanation part has links that are wrong.

cc @jon-tow

**Comment by AndrewGodfrey** [comment]
Yes, now you mention it, it was probably copied from a page describing the other license, but not updated properly. As dylancvdean says, it should also mention the ShareAlike obligation (if it’s going to list obligations at all).
